src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.cs

```diff
@@ -926,14 +924,14 @@ public class C
     {{
     }}
 }}"
-                    }
+                    },
+                    AnalyzerConfigFiles = { ("/.editorconfig", $"[*]{Environment.NewLine}{editorConfigText}") }
```

Comment:
IMO, this deserves a static helper method to return `AnalyzerConfigFiles` given `string editorConfigText` as the input.

Reply:
@mavasani For this specific test file, I used it only twice (in the same test). Did you mean to introduce it in a new helper class and use it from all tests?

Reply:
The form used here is the form I generally expect to see.